commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -41,13 +45,34 @@
       arity = "1")
   private boolean slashingProtectionEnabled = false;
 
+  @Option(
```

Comment:
may I also recommend a cli option for JDBC Driver class? Which can be statically load using Class.forName